src/NServiceBus.Metrics/Performance/PerformanceStatisticsMetricBuilder.cs

```diff
@@ -0,0 +1,24 @@
+using Metrics;
+using NServiceBus.Features;
+
+class PerformanceStatisticsMetricBuilder : IMetricBuilder
+{
+    public void WireUp(FeatureConfigurationContext featureConfigurationContext, MetricsContext metricsContext, Unit messagesUnit)
+    {
+        var messagesPulledFromQueueMeter = metricsContext.Meter("Messages Pulled from Queue", messagesUnit);
```

Comment:
@mikeminutillo - Regarding your question:

>Do the names of the receive perf stats need to match the eventual perf counters?

I think at least for the first version, it would be best if these names matched what we currently have in the performance counters, as they're basically 1 to 1. 

Ran it by @kbaley:

Currently, in the performance counters we have:
```
    public const string MessagesPulledPerSecondCounterName = "# of msgs pulled from the input queue /sec";
    public const string MessagesProcessedPerSecondCounterName = "# of msgs successfully processed / sec";
    public const string MessagesFailuresPerSecondCounterName = "# of msgs failures / sec";
```

But I think we can tweak it to the below: 

"# of messages pulled from the input queue / sec";
"# of messages successfully processed / sec";
"# of message failures / sec";